tools/kail.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mkdir -p kail
+curl -L https://github.com/boz/kail/releases/download/v0.6.0/kail_0.6.0_linux_amd64.tar.gz \
```

Comment:
what are your thoughts regarding Windows support? (out of this PR's scope obviously)

Reply:
could we already set a convention like tools/${TOOL_NAME}_${OS}.sh is released in different OS flavors?

Reply:
My tendency is to think that each tool should detect the environment it needs during the install. How specific the platform needs to be match will depend on the tool.

Let's get this merged and then regroup on windows.

Reply:
👍